MobileSDK_To_AmplifySwift_Guidance.md

```diff
@@ -0,0 +1,162 @@
+| Transfer with Metadata                                                                             | Supported (`StorageUploadFileRequest.Options.metadata`) |
+| Background Transfers                                                                               | Not currently supported in Amplify Swift ([see SDK doc](https://docs.amplify.aws/gen1/swift/sdk/storage/transfer-utility/#background-transfers)) |
+
+---
+
+### REST API
+
+(Replace **AWSAPIGatewayClient**)
+
+| AWS Mobile SDK for iOS                                      | Amplify Swift                                                                                                                                                                                                                                                                                                                                                           |
+| ----------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Invoke](https://docs.amplify.aws/gen1/swift/sdk/api/rest/) | [Create](https://docs.amplify.aws/gen1/swift/build-a-backend/restapi/set-up-rest-api/#make-a-post-request) / [Fetch](https://docs.amplify.aws/gen1/swift/build-a-backend/restapi/fetch-data/) / [Update](https://docs.amplify.aws/gen1/swift/build-a-backend/restapi/update-data/) / [Delete](https://docs.amplify.aws/gen1/swift/build-a-backend/restapi/delete-data/) |
+| [IAM Auth](https://docs.amplify.aws/gen1/swift/sdk/api/rest/#iam-authorization)                                                    | [Configure IAM](https://docs.amplify.aws/gen1/swift/build-a-backend/restapi/customize-authz/#iam-authorization)                                                                                                                                                                                                                                                         |
+| [Cognito User Pools](https://docs.amplify.aws/gen1/swift/sdk/api/rest/#cognito-user-pool-authorization)                                          | [CUP Auth](https://docs.amplify.aws/gen1/swift/build-a-backend/restapi/customize-authz/#cognito-user-pool-authorization)                                                                                                                                                                                                                                                |
```

Comment:
first link : https://docs.amplify.aws/gen1/swift/sdk/api/rest/#cognito-user-pools-authorization

missing s character

Reply:
we can probably mention Cognito User Pools Auth instead of CUP auth to not confuse customers.